netket/hilbert/random/continuous.py

```diff
@@ -0,0 +1,21 @@
+import jax
+from jax import numpy as jnp
+
+from netket.hilbert import Particles
+from netket.utils.dispatch import dispatch
+
+
+@dispatch
+def random_state(hilb: Particles, key, batches: int, *, dtype):
+    """Positions particles w.r.t. normal distribution if no periodic boundary conditions are applied in a spatial dimension.
+    Otherwise the particles are positioned evenly along the box from 0 to L, with Gaussian noise."""
+    gaussian = jax.random.normal(key, shape=(batches, hilb.size))*0.2
```

Comment:
what is this 0.2? can we do something more generic?

Reply:
it is the noise level but since the signature is random_state(self, key, size=None, dtype=np.float32)
there is no place for another parameter to pass.

Reply:
Yeah, but I mean: why did you pick 0.2? Is it possible to chose a value that depends on the Hilbert space (say, for example, on box size times the number of particles, or anything)

And, if not, at least document it that this 0.2 was chosen completely arbitrarily.